test_logwarn

```diff
@@ -98,6 +98,7 @@ is-ignored '\[warn\].* fatal: Invalid revision range .*\.\.'
 is-ignored '\[warn\] \[pid:[0-9]*\] $'
 is-ignored '\[error\].*cmd returned 32768$'
 is-ignored '\[error\] Worker 17519 has no heartbeat'
+is-ignored '\[error\] \[pid:4360\] Publishing opensuse\.openqa\.comment\.create failed: Can\'t connect: System error \(event ID: 726555, job ID: 4926463, 9 attempts left\)'
```

Comment:
So this was supposed to match this:

    [2025-03-16T06:47:19.984452Z] [error] [pid:4360] Publishing opensuse.openqa.comment.create failed: Can't connect: System error (event ID: 726555, job ID: 4926463, 9 attempts left)

I tried to simplify it. Even trying to remove more detail here the tests don't seem to pass.

Reply:
Did you miss
```
'!\[error\].*
```

look at the previous examples. Maybe that?

Reply:
Not in the "test". But thanks